### PR TITLE
Launchpad - Update launchpad site preview height for desktop viewport

### PR DESCRIPTION
**Pull request by agrullon95**
Related to #74651

## Test/Review Estimation

Test: short
Review: short

## Proposed Changes

* Update Launchpad site preview height to take up entire viewport

**Before:**

CleanShot 2023-04-06 at 10 09 23@2x

**After:**
CleanShot 2023-04-06 at 10 08 22@2x

## Testing Instructions

* Checkout this branch (or use calypso live link)
* Create or use existing launchpad enabled site
* Navigate to launchpad
* Use the browser zoom feature to modify the browser screen size
* Verify the Launchpad site preview height expands/shrinks to match the viewport height
* Verify the mobile view is the same as production
* Verify the changes don't affect anything else besides the site preview height on other flows as welll(You can modify the URL to change the flow . `/setup/newsletter/launchpad`, `/setup/link-in-bio/launchpad`, etc.)

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by edanzer** [comment]
UPDATE: Sounds like this should affect all flows, but should only affect preview height for desktop previews, which is what I'm seeing. 

@agrullon95 This is testing mostly as expected for me. 

I tested free, write, build, newsletter, link-in-bio, and newsletter flows. All of them seem to be equally affected by this change - desktop preview expands with viewport height, mobil preview does not. 

In your testing instruction you include "Verify other flows have not been affected". I was wondering if could get clarification on that. From what I can tell, all flows are affected in the same way. Are there some that should not be?

File: client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { StepNavigationLink } from 'calypso/../packages/onboarding/src';
 import Badge from 'calypso/components/badge';
 import ClipboardButton from 'calypso/components/forms/clipboard-button';
 import Tooltip from 'calypso/components/tooltip';
-import WordPressLogo from 'calypso/components/wordpress-logo';
```
Comment (edanzer) [inline client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx:9]: I had a question about why we're moving the WordPressLogo in this PR. Sounds like it is needed due to the other styling changes.

**Review by edanzer — APPROVED** [review]

File: client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss [hunk 2]
```diff
@@ -28,12 +28,19 @@
 		}
 	}
 
+	.step-container__content {
+		@include break-large {
+			height: 100vh;
```
Comment (jeyip) [inline client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss:33]: If I understand things correctly, it seems like the above style, [this](https://github.com/Automattic/wp-calypso/pull/75129/files#diff-3f2ed61b57c0fdab5f9b1914740a42b1352e8035d490bc5a318d6950c639197dR43) style, or [this](https://github.com/Automattic/wp-calypso/pull/75129/files#diff-3f2ed61b57c0fdab5f9b1914740a42b1352e8035d490bc5a318d6950c639197dR473) style is doing the bulk of the heavy lifting with the responsive iframe preview. Is that right?

**Comment by jeyip** [comment]
> Sounds like this should affect all flows, but should only affect preview height for desktop previews, which is what I'm seeing.

Good clarification. Just want to confirm this is what we expect? @agrullon95

**Comment by jeyip** [comment]
## Testing
### Requirements
- [x] Verify the Launchpad site preview height expands/shrinks to match the viewport height
- [x] Verify the mobile view is the same as production
- [x] Verify the changes don't affect anything else besides the site preview height on other flows as welll(You can modify the URL to change the flow . /setup/newsletter/launchpad, /setup/link-in-bio/launchpad, etc.)

### Browsers
- [x] Chrome

**Review by jeyip — APPROVED** [review]

**Review by nuriapenya — COMMENTED** [review]
It looks & works perfectly 👌🏽  thanks @agrullon95

**Comment by agrullon95** [comment]
> Sounds like this should affect all flows, but should only affect preview height for desktop previews, which is what I'm seeing.

@edanzer / @jeyip - That's correct. I meant that say that these changes should only affect the site preview height on all flows.

**Pull request merged**